stan/math/prim/scal/fun/inv_Phi.hpp

```diff
@@ -0,0 +1,98 @@
+#ifndef STAN_MATH_PRIM_SCAL_FUN_INV_PHI_HPP
+#define STAN_MATH_PRIM_SCAL_FUN_INV_PHI_HPP
+
+#include <boost/math/tools/promotion.hpp>
+#include <boost/math/special_functions/erf.hpp>
+#include <stan/math/prim/scal/fun/constants.hpp>
+#include <stan/math/prim/scal/err/check_not_nan.hpp>
+#include <stan/math/prim/scal/err/check_bounded.hpp>
+#include <stan/math/prim/scal/fun/Phi.hpp>
+#include <stan/math/prim/scal/fun/log1m.hpp>
+
+namespace stan {
+  namespace math {
+
+    /**
+     * The inverse of the unit normal cumulative distribution function.
+     *
+     * The return value for a specified input probability, $p$, is
+     * the unit normal variate, $x$, such that
+     *
+     * \f$\Phi(x) = \int_{-\infty}^x \mbox{\sf Norm}(x|0, 1) \ dx = p\f$
+     *
+     * Algorithm first derived in 2003 by Peter Jon Aklam at
+     * http://home.online.no/~pjacklam/notes/invnorm/
+     *
+     * @param p Argument between 0 and 1.
+     * @return Real number
+     */
+    template <typename T>
+    inline typename boost::math::tools::promote_args<T>::type
+    inv_Phi(const T p) {
+      // overridden in fvar and var, so can hard-code
```

Comment:
If it's overridden, replace `const T p` with `double p` (no `const` necessary);  if it must be a template param for some reason, it should be a constant reference.